ansible/roles_ocp_workloads/ocp4_workload_quay_workshop/tasks/remove_workload.yml

```diff
@@ -0,0 +1,15 @@
+---
+# Implement your workload removal tasks here
+# ------------------------------------------
+
+
+
+
+
```

Comment:
I believe yamllint will complain that there are too many blank lines here. To test, run `yamllint` with the current directory as `roles_ocp_workloads`.